src/components/FormElementLabelContainer.tsx

```diff
@@ -28,7 +28,7 @@ function FormElementLabelContainer({
           {element.label}
         </label>
         {element.hint && (
-          <Tooltip title={element.hint} arrow>
+          <Tooltip title={element.hint} arrow enterTouchDelay={0}>
```

Comment:
Want to change the other tooltip(s ?) as well? Think theres one in the `BaseForm` component

Reply:
Agreed

Reply:
Actually, disagreed. It is on a button and a press on that should not show the tooltip, it should just execute the action. IMO

Reply:
Sounds good